### DISCUSSION TITLE
How to have RepeatableEntry content to take up full width?

### DISCUSSION THREAD

**Discussion by NgYueHong** [filamentphp/filament#12430] [Help]
### Package

Panel builder

### Package Version

3.2

### How can we help you?

This is before the label is hidden:
![image](https://github.com/filamentphp/filament/assets/6602432/2c017da3-13fa-41c3-8df0-7b8b8ea8a479)

This is after the label is hidden:
![image](https://github.com/filamentphp/filament/assets/6602432/ba36365f-e936-4e14-ac8f-5dc95bfa5505)

How can I have the RepeatableEntry content to take up full width after the label is removed?

**Comment by leandrocfe** [comment]
columnSpanFull()
?

**Reply by NgYueHong** [reply]
Already applied, but it doesn't make any difference.

**Reply by leandrocfe** [reply]
Please share the code you are using

**Reply by NgYueHong** [reply]
@leandrocfe Thank you for following up. The problem is solved after I put `->inlineLabel(false)` to `RepeatableEntry`.
